packages/react-tracking-pixel/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@shopify/react-tracking-pixel",
+  "version": "0.1.0",
+  "license": "MIT",
+  "description": "Allows react apps to easily embed tracking pixel iframes",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc --p tsconfig.build.json",
+    "prepublishOnly": "yarn run build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "author": "Shopify Inc.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/quilt.git"
+  },
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
+  },
+  "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-tracking-pixel/README.md",
+  "dependencies": {
+    "@shopify/react-preconnect": "^0.1.2"
+  },
+  "devDependencies": {
+    "typescript": "~3.0.1",
+    "enzyme": "^3.3.0"
+  },
+  "peerDependencies": {
+    "react": "^16.4.1"
+  }
+}
```

Comment:
If I preview what is going to be uploaded to the registry, I see a few unnecessary files:

```
➜ npm pack
> shopify-react-tracking-pixel-0.1.0.tgz
➜ tar -tf shopify-react-tracking-pixel-0.1.0.tgz
> package/package.json
> package/README.md
> package/tsconfig.build.json
> package/dist/index.d.ts
> package/dist/index.js
> package/dist/TrackingPixel.d.ts
> package/dist/TrackingPixel.js
> package/src/index.ts
> package/src/tests/TrackingPixel.test.tsx
> package/src/TrackingPixel.scss
> package/src/TrackingPixel.tsx
```

You can use [`package.json#files`](https://docs.npmjs.com/files/package.json#files) to weed out `src` / `tsconfig` stuff.

Reply:
Changed the files included: 
`package/package.json
package/README.md
package/dist/index.d.ts
package/dist/index.js
package/dist/TrackingPixel.d.ts
package/dist/TrackingPixel.js`